packages/date-time-picker/test/benchmark/basic-test.ts

```diff
@@ -9,10 +9,10 @@ the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTA
 OF ANY KIND, either express or implied. See the License for the specific language
 governing permissions and limitations under the License.
 */
-import '@spectrum-web-components/time-field/sp-time-field.js';
+import '@spectrum-web-components/date-time-picker/sp-date-time-picker.js';
 import { html } from '@spectrum-web-components/base';
 import { measureFixtureCreation } from '../../../../test/benchmark/helpers.js';
 
 measureFixtureCreation(html`
-    <sp-time-field></sp-time-field>
+    <sp-date-time-picker></sp-date-time-picker>
```

Comment:
Please configure this picker with some defaults to better exhibit the performance profile of the element.